src/components/input-field/examples/input-field-number.tsx

```diff
@@ -32,6 +32,11 @@ export class InputFieldNumberExample {
     }
 
     public render() {
+        let formatLabel = 'Format number';
+        if (this.formatNumber) {
+            formatLabel = 'Unformat number';
+        }
```

Comment:
Personally, I don't think this is any better than the ternary that was removed (I actually think this is ever so slightly worse, because it's removed from the context where it's used), but if this is the price we have pay for getting rid of some of the much worse ternary-uses, then I guess it's a price I'm willing to have us pay… 😅

Reply:
Yeah my thoughts exactly :sweat_smile: 
There might be some cases where it would make sense to have it on multiple lines, but as I said it's far too easy to abuse and we have some horrible examples of that :smile: